src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -9,7 +9,7 @@ import {convertString} from '../../../utils/string/StringUtils';
 import {colors} from '../../../utils/colors';
 
 const TopicDomainHeader = (props) => {
-  const {handleOnMemberPress, hideSeeMember, isFollow, memberCount, topicDetail} = props;
+  const {domain, handleOnMemberPress, hideSeeMember, isFollow, memberCount} = props;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The prop `topicDetail` has been replaced with `domain`. Ensure that all instances where this component is used have been updated to pass the `domain` prop instead of `topicDetail`. Also, make sure that the `domain` prop is being correctly derived from the state or props in the parent components.



<!-- This is an auto-generated comment by OSS CodeRabbit -->